src/client/clientmap.cpp

```diff
@@ -180,17 +199,413 @@ void ClientMap::getBlocksInViewRange(v3s16 cam_pos_nodes,
 			p_nodes_max.Z / MAP_BLOCKSIZE + 1);
 }
 
+class MapBlockFlags
+{
+public:
+	static constexpr u16 CHUNK_INNER_BITS  = 4; // number of bits to address inside a chunk
+	static constexpr u16 CHUNK_EDGE = 16; // 2 ^ CHUNK_SHIFT
```

Comment:
Will this work with MAP_BLOCKSIZE > 16?

Reply:
These are chunks of mapblocks. Will work with any mapblock size.